src/main/java/frc/robot/Constants.java

```diff
@@ -47,24 +47,24 @@
      
     //SHOOTER RPM (max=7000rpm min=100) (max enc units per 100ms = 18000) ***max changes with tuning...20000 now
     
-    public static final NTValue TEST_UPPER = new NTValue(10000, "Shooter Test Upper");
-    public static final NTValue TEST_LOWER = new NTValue(0, "Shooter Test Lower");
+    public static final NTValue TEST_UPPER = new NTValue((double) 10000, "Shooter Test Upper");
```

Comment:
I don't think you have to cast it when it is a parameter, I think it will read it as a double because the parameter accepts a double. If it wasn't working, it's probably because you used the uppercase `Double` class instead of the `double` type in the parameter in the constructor

```suggestion
    public static final NTValue TEST_UPPER = new NTValue(10000, "Shooter Test Upper");
```